src/Tizen.NUI.Scene3D/src/ModelView.cs

```diff
@@ -76,58 +70,74 @@ public ModelView(ModelView modelView) : this(Interop.ModelView.NewModelView(Mode
         }
 
         /// <summary>
-        /// Assignment operator.
+        /// Retrieves model root View.
         /// </summary>
-        /// <param name="modelView">Handle to an object.</param>
-        /// <returns>Reference to this.</returns>
         // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public ModelView Assign(ModelView modelView)
+        public View ModelRoot
         {
-            ModelView ret = new ModelView(Interop.ModelView.ModelViewAssign(SwigCPtr, ModelView.getCPtr(modelView)), false);
-            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
-            return ret;
+            get
+            {
+                View ret = new View(Interop.ModelView.GetModelRoot(SwigCPtr), false);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+                return ret;
```

Comment:
could you please check this message?

get_ModelRoot creates an exception of type Exception, an exception type that should not be raised in a property. If this exception instance might be raised, use a different exception type, convert this property into a method, or change this property's logic so that it no longer raises an exception.

Reply:
Thank you I added a method.